pyspades/player.py

```diff
@@ -172,6 +173,10 @@ def loader_received(self, loader: enet.Packet) -> None:
             return
         call_packet_handler(self, loader)
 
+    @register_packet_handler(loaders.ProtocolExtensionInfo)
+    def on_new_player_recieved(self, contained: loaders.ProtocolExtensionInfo) -> None:
```

Comment:
Should be `on_ext_info_recieved` or something right?

Reply:
oh, crap, thanks for spotting that.